RecoTauTag/RecoTau/plugins/PATTauDiscriminationAgainstElectronMVA6.cc

```diff
@@ -104,7 +112,7 @@ double PATTauDiscriminationAgainstElectronMVA6::discriminate(const TauRef& theTa
 	double deltaREleTau = deltaR(theElectron.p4(), theTauRef->p4());
 	deltaRDummy = deltaREleTau;
 	if( deltaREleTau < 0.3 ){ 	
-	  double mva_match = mva_->MVAValue(*theTauRef, theElectron, usePhiAtEcalEntranceExtrapolation_);	  
+	  double mva_match = mva_->MVAValue(*theTauRef, theElectron, usePhiAtEcalEntranceExtrapolation_, bField_);	  
```

Comment:
why not set bField in the mva::beginEvent call ?
It seems like a simpler solution with fewer changes in interfaces.